src/routes/oath.ts

```diff
@@ -0,0 +1,83 @@
+import { Router } from 'express'
+require('dotenv').config();
+import axios from 'axios';
+import querystring from 'querystring';
+
+
+
+const router = Router()
+
+// env constants
+const CLIENT_ID = process.env.CLIENT_ID;
+const CLIENT_SECRET = process.env.CLIENT_SECRET;
+const REDIRECT_URI = process.env.REDIRECT_URI;
+const AUTH_URL = process.env.AUTH_URL;
```

Comment:
Build the auth url in the program instead of hard code in env. Since it auth url builds off of client id and redirect url only one place to change either.

Reply:
Updated!